e2e/src/usecases/PINChange.js

```diff
@@ -23,7 +23,9 @@ export default ChangePIN = () => {
 
     // Enter an invalid pin and check that we get the correct error and start over
     await enterPinUi('902100')
-    await expect(element(by.text("The PINs didn't match"))).toBeVisible()
+    await waitFor(element(by.text("The PINs didn't match")))
+      .toBeVisible()
+      .withTimeout(15 * 1000)
```

Comment:
Detox expected a timeout on waitFor calls, it will only wait up to 15 seconds in this case, and continue on if found sooner: [docs](https://github.com/nubank/detox/blob/master/docs/APIRef.waitFor.md#tobevisible).